### DISCUSSION TITLE
Not working with new pixel 6a

### DISCUSSION THREAD

**Discussion by Davidje1982** [AppGround-io/bluetooth-keyboard-and-mouse-support#287] [Bugs & Problems]
Hi all,

I used to have an old Motorola moto G5 plus with arrow os (android 10).
I used this app with no problem connecting to my rpi4+ running twister os.

Since my Moto g crashed, I have a Google pixel 6a with android 13.
I am unable to connect them with this app...
I am connected via Bluetooth but the app is not connecting to the rpi.

On my android device it says that my bluetooth connection is an audio connection. This can not be the problem, since this was the same on my Moto G.

Anyone any suggestions or information on how to solve this issue?

Thank you in advance.
Kind regards,
David

**Comment by AppGround-io** [comment]
This sounds like the devices are not correctly paired with each other. Please remove the Bluetooth pairing on both devices and connect again through the connection screen in the app.

**Comment by Davidje1982** [comment]
I found out yesterday what the problem is...

It is this:
https://github.com/RPi-Distro/bluez-firmware/issues/9

The only thing is that I tried the test solution and I didn't get it to work.
Removing Bluetooth pairing and rebooting both systems did not help...
I also would prefer an official solution.

Any help with this would be great.

Thank you in advance.



________________________________
Van: AppGround-io ***@***.***>
Verzonden: donderdag 16 maart 2023 4:32
Aan: AppGround-io/bluetooth-keyboard-and-mouse-support ***@***.***>
CC: Davidje1982 ***@***.***>; Author ***@***.***>
Onderwerp: Re: [AppGround-io/bluetooth-keyboard-and-mouse-support] Not working with new pixel 6a (Discussion #287)


This sounds like the devices are not correctly paired with each other. Please remove the Bluetooth pairing on both devices and connect again through the connection screen in the app.
